### DISCUSSION TITLE
How do I add static text on the list page, before or after the meeting list?

### DISCUSSION THREAD

**Discussion by troybon** [code4recovery/12-step-meeting-list#488] [Q&A] [answered]
Hello, 

I am new to this plugin and was wondering how I add static text  area on the list page, before or after the meeting list? 

Thanks,
Troy

**Comment by joshreisner** [comment, marked as answer]
Hi Troy,

You can use widgets at Appearance > Widgets to add, for example, a text widget. We generally recommend below the meeting list, because remember that most of your users are using a mobile device, so adding text above pushes the interface down.

![Screen Shot 2021-08-03 at 10 36 18 PM](https://user-images.githubusercontent.com/1551689/128127641-22d8d3cd-fd10-473f-9e8a-bce37a338df9.png)
![Screen Shot 2021-08-03 at 10 35 30 PM](https://user-images.githubusercontent.com/1551689/128127646-0b76af83-4417-415b-b8d9-c97c3830ceae.png)

**Comment by troybon** [comment]
Thank you very much.

Troy M. Bon, MBA
770.509.0022

On Aug 4, 2021, at 1:37 AM, Josh Reisner ***@***.***> wrote:

﻿

Hi Troy,

You can use widgets at Appearance > Widgets to add, for example, a text widget. We generally recommend below the meeting list, because remember that most of your users are using a mobile device, so adding text above pushes the interface down.

[Screen Shot 2021-08-03 at 10 36 18 PM]<https://user-images.githubusercontent.com/1551689/128127641-22d8d3cd-fd10-473f-9e8a-bce37a338df9.png>
[Screen Shot 2021-08-03 at 10 35 30 PM]<https://user-images.githubusercontent.com/1551689/128127646-0b76af83-4417-415b-b8d9-c97c3830ceae.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/code4recovery/12-step-meeting-list/discussions/488#discussioncomment-1128706>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVCQMN5FX4F5LPB5IKEAPD3T3DGY7ANCNFSM5BP23MUA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.
